Index: stage0/rootfs/etc/bash.bashrc
===================================================================
--- stage0.orig/rootfs/etc/bash.bashrc
+++ stage0/rootfs/etc/bash.bashrc
@@ -29,13 +29,13 @@ PS1='${debian_chroot:+($debian_chroot)}\
 #esac

 # enable bash completion in interactive shells
-#if ! shopt -oq posix; then
-#  if [ -f /usr/share/bash-completion/bash_completion ]; then
-#    . /usr/share/bash-completion/bash_completion
-#  elif [ -f /etc/bash_completion ]; then
-#    . /etc/bash_completion
-#  fi
-#fi
+if ! shopt -oq posix; then
+  if [ -f /usr/share/bash-completion/bash_completion ]; then
+    . /usr/share/bash-completion/bash_completion
+  elif [ -f /etc/bash_completion ]; then
+    . /etc/bash_completion
+  fi
+fi

 # if the command-not-found package is installed, use it
 if [ -x /usr/lib/command-not-found -o -x /usr/share/command-not-found/command-not-found ]; then
@@ -53,3 +53,9 @@ if [ -x /usr/lib/command-not-found -o -x
                fi
        }
 fi
+
+# load system-wide alias definitions
+if [ -f /etc/bash.bash_aliases ]; then
+  . /etc/bash.bash_aliases
+fi
+

